src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -6,14 +6,15 @@
 import * as vscode from 'vscode';
 import * as path from 'path';
 import * as keytarType from 'keytar';
-import { MongoClient, ReplSet } from "mongodb";
+import { MongoClient, ReplSet, connect } from "mongodb";
```

Comment:
Why'd ya add 'connect'? Actually the build might be failing because of this 'unused variable'

Reply:
It has a mind of its own...